app/services/email_due_soon_and_overdue_defects.rb

```diff
@@ -6,7 +6,8 @@ def initialize(defects:)
   end
 
   def call
-    DueSoonAndOverdueDefectsMailer.notify(
+    DefectsMailer.notify(
```

Comment:
This threw me a bit, can we have two mailer actions that share functionality instead? So we can have `DefectsMailer.due_soon_and_overdue` and `DefectsMailer.escalated`? Then all you have to do is set the template ID and then call a shared method.